SimPEG/Directives.py

```diff
@@ -509,8 +542,8 @@ class Update_IRLS(InversionDirective):
     f_min_change = 1e-2
     beta_tol = 5e-2
     prctile = 95
-    chifact = 1.
-    l2model = None
+    chifact_start = 1.
```

Comment:
what is the distinction between the `chifact_start` and `chifact_target`? @fourndo?

Reply:
chifact_start allows you to start the IRLS before reaching the end of the l2 inversion. chifact_target is the same as for the betacooling directive.

Reply:
ah, kk. what about `chifact_l2`? to indicate it is on the l2 inversion?

Reply:
@fourndo: thoughts on naming here?